test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -176,6 +176,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.apm.serviceMapEnabled (boolean)',
         'xpack.apm.ui.enabled (boolean)',
         'xpack.apm.ui.maxTraceItems (number)',
+        'xpack.apm.managedServiceUrl (string)',
```

Comment:
This URL may be publicly accessible when exposed in this manner. Is this URL considered public knowledge, or is this information that should be restricted to authorized users?

Reply:
@legrego I am not 100% sure if i understand it when you publicly accessible. So the idea is, this URL has to be exposed to users as part of APM Serverless Onboarding tutorials. They will configure this URL in their applications to send data. Also this URL is protected with API Keys. 

The understanding is, only logged in users in Kibana will be able to see this URL because it will only be displayed during the Serverless Tutorial.

Reply:
Sorry, by "publicly accessible", I mean it's available to _anyone_ on the public Internet.

When config entries are exposed to the browser in this fashion, they are potentially included on _unauthenticated_ pages, such as the login screen. This means that anyone on the public Internet can scrape this information without providing credentials -- the security team is a codeowner on this test to ensure that we don't leak anything sensitive here.

> The understanding is, only logged in users in Kibana will be able to see this URL because it will only be displayed during the Serverless Tutorial.

I recommend having the Serverless Tutorial request this information via an API call instead, since API calls can only be made by logged in users.

Reply:
@achyutjhunjhunwala Please check with the apm server folks if the managed server url is unique for each user, or public knowledge.

Reply:
There is no more sensitive information in the URL than there is in the Kibana URL. It's `https://<project-id>.apm.<suffix>`, where `<suffix>` is exactly the same as for Kibana and Elasticsearch. If users know the Kibana URL, it's fine for them to also know the APM URL.

Reply:
@legrego I guess some users may consider it sensitive, but I wouldn't think so. I meant to mention before that the project ID is also in the Kibana URL - so the users who can see the the project ID for the APM intake URL already have it.

Reply:
Good enough for me, thanks again @axw